www/v1/stock2shop/dal/channels/example/Products.php

```diff
@@ -0,0 +1,470 @@
+<?php
+
+namespace stock2shop\dal\channels\example;
+
+use stock2shop\dal\channel\Products as ProductsInterface;
+use stock2shop\vo;
+use stock2shop\helpers;
+
+/**
+ * Products
+ *
+ * This class is where the Data Access Layer is mapped onto
+ * the Stock2Shop Value Objects from the source system you are
+ * integrating with.
+ *
+ * @package stock2shop\dal\example
+ */
+class Products implements ProductsInterface
+{
+    /** @const string DATA_PATH */
+    const DATA_PATH = __DIR__ . "/data";
+
+    /** @const string CHANNEL_SEPARATOR_VARIANT */
+    const CHANNEL_SEPARATOR_VARIANT = "variant_separator";
+
+    /** @const string CHANNEL_SEPARATOR_IMAGE */
+    const CHANNEL_SEPARATOR_IMAGE = "image_separator";
+
+    /**
+     * Sync
+     *
+     * This method synchronises products, variants and images on the channel.
+     *
+     * - product.id is the file name for the product.
+     * - product.variant[].channel_variant_code is the file name for a variant.
+     * - product.image[].channel_image_code is the file name for an image.
+     *
+     * @param vo\ChannelProduct[] $channelProducts
+     * @param vo\Channel $channel
+     * @param vo\Flag[] $flagsMap
+     * @return ChannelProduct[] $channelProducts
+     */
+    public function sync(array $channelProducts, vo\Channel $channel, array $flagsMap): array
+    {
+
+        /** @var string $imageSeparator Channel separator meta for images. */
+        $imageSeparator = helpers\Meta::get($channel->meta, self::CHANNEL_SEPARATOR_IMAGE);
+
+        /** @var string $variantSeparator Channel separator meta for variants. */
```

Comment:
perhaps a comment explaining how any custom meta can be created and used in a channel?

Reply:
Added to example\Products. Also added another example of using channel meta to make the connector's configurable.